cdap-explore/src/main/java/co/cask/cdap/explore/guice/ExploreRuntimeModule.java

```diff
@@ -309,7 +312,10 @@ public boolean accept(String className, URL classUrl, URL classPathUrl) {
     // we need to add this, otherwise when hive runs a MapRedLocalTask it cannot find
     // "org.apache.hadoop.hive.serde2.SerDe" class in its classpath.
     List<String> orderedDependenciesWithHiveJar = Lists.newArrayList(orderedDependenciesStr);
-    orderedDependenciesWithHiveJar.add(new HiveConf().getJar());
+    String hiveExecJar = new JobConf(Task.class).getJar();
+    Preconditions.checkNotNull(hiveExecJar, "Couldn't locate hive-exec.jar to be included in HADOOP_CLASSPATH " +
```

Comment:
log the hiveExecJar as a debug message, so we know what is the jar we are adding when we want to debug.